docs/source/create/entities/web-entity.rst

```diff
@@ -0,0 +1,130 @@
+
++------------------------+------------------------------------------------------------------+--------------------------+
+| Video codec            | Comments                                                         | Supported containers     |
++========================+==================================================================+==========================+
+| VP9 (recommended)      | Good quality, slow encoding                                      | WebM, Matroska           |
++------------------------+------------------------------------------------------------------+--------------------------+
+| VP8                    | Recommended if your APU has hardware accelerated encoding for    | WebM, Matroska           |
+|                        | VP8, but not VP9. Slightly faster than VP9.                      |                          |
+|                        | Slightly lower quality to compression ratio than VP9.            |                          |
++------------------------+------------------------------------------------------------------+--------------------------+
+| Theora                 | Low quality, low compression, outdated                           | Matroska                 |
++------------------------+------------------------------------------------------------------+--------------------------+
+
++------------------------+------------------------------------------------------------------+--------------------------+
+| Audio codec            | Comments                                                         | Supported containers     |
++========================+==================================================================+==========================+
+| Opus                   | Recommended, works well with high and low bitrate                | WebM, Matroska, Ogg      |
++------------------------+------------------------------------------------------------------+--------------------------+
+| Vorbis                 | Lower quality to compression ratio than Opus                     | WebM, Matroska, Ogg      |
++------------------------+------------------------------------------------------------------+--------------------------+
+| FLAC                   | Lossless, bigger files                                           | Matroska, Ogg            |
++------------------------+------------------------------------------------------------------+--------------------------+
+| MP3                    | Lower quality to compression ratio than Opus and Vorbis          | Matroska                 |
++------------------------+------------------------------------------------------------------+--------------------------+
+
+--------------------------
+Reencode media with ffmpeg
+--------------------------
+
+Here are some example commands for reencoding video files using ffmpeg:
+
++------------------------+------------------------------------------------------------------+--------------------------+
```

Comment:
This table isn't displaying in the compiled page output. It just looks like the table formatting doesn't have all the vertical bars etc. lined up.